lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -487,6 +487,10 @@
             - "C(user_migratable) - Allow manual migration only."
             - "If no value is passed, default value is set by oVirt/RHV engine."
         version_added: "2.5"
+    ticket:
+        description:
+            - "If I(true), returns to vms remote_vv_file ticket to remote-viewer."
```

Comment:
If I(true), in addition return I(remote_vv_file) inside I(vm) dictionary, which contains compatible content for remote-viewer application.

Reply:
Please fix this comment and we can merge.